mne/io/egi/egimff.py

```diff
@@ -9,15 +9,18 @@
 
 import numpy as np
 
+from mffpy import Reader
```

Comment:
This should be an optional dependency, which means you should nest the import inside the function that uses it (see how we treat other optional dependencies like nibabel, dipy, sklearn, etc.)